src/main/java/com/google/api/codegen/config/GapicInterfaceConfig.java

```diff
@@ -86,6 +86,11 @@ public String getName() {
     return hasInterfaceNameOverride() ? getInterfaceNameOverride() : getInterface().getSimpleName();
   }
 
+  @Override
+  public String getRawName() {
+    return getInterface().getSimpleName();
+  }
+
```

Comment:
It's used by two functions in `SurfaceNamer`.

I think it's a useful abstraction - there is also a `getName()` function. This is an override of the method in `InterfaceConfig` so that DiscoGapicConfig can have its own implementation.